scripts/scil_analyse_lesions_load.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+This script will output informations about lesion load in bundle(s).
+The input can either be streamlines, binary bundle map, or a bundle voxel
+label map.
```

Comment:
Can you say 1-2 sentences about the min_lesion_volume parameter and how it comes into the script/algorithm please?